demo/api/src/app.module.ts

```diff
@@ -55,13 +56,22 @@ export class AppModule {
             imports: [
                 ConfigModule.forRoot(config),
                 DbModule,
-                GraphQLModule.forRootAsync({
+                GraphQLModule.forRootAsync<ApolloDriverConfig>({
                     driver: ApolloDriver,
                     imports: [BlocksModule],
                     useFactory: (dependencies: Record<string, unknown>) => ({
                         debug: config.debug,
                         playground: config.debug,
                         autoSchemaFile: "schema.gql",
+                        introspection: process.env.NODE_ENV === "development",
```

Comment:
Apollo disables GraphQL introspection in production by default, so I don't think this is necessary 🤔

Reply:
I'd disable it for test or staging also

Reply:
With "production" I mean `process.env.NODE_ENV="production"`, so all environments are included

Reply:
> The default value is true, unless the NODE_ENV environment variable is set to production.

https://www.apollographql.com/docs/apollo-server/api/apollo-server/#introspection

Reply:
Yeah but it should be set to false for all envs except dev. If we leave the default it would be true for eg staging or test envs

Reply:
process.env.NODE_ENV="production" is true for dev, test and staging environments in our setup (and probably most setups).

Reply:
```suggestion
```

As discussed internally, we can remove this because it's Apollo's default behavior